eng/testing/AppleRunnerTemplate.sh

```diff
@@ -1,31 +1,24 @@
 ﻿#!/usr/bin/env bash
```

Comment:
This file seems to be doing what we already have in Helix SDK:
https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Helix/Sdk/tools/xharness-runner/ios-helix-job-payload.sh
and which makes sure things don't time out and results get published back to AzDO

Reply:
we still need a way to run locally though :)

Reply:
Fair point. For that this script would suffice since things are easier locally

Reply:
But we have to find a way to maintain it. Maybe we should enable support for local story in the SDK

Reply:
I think it's fine and perhaps also useful to have different script for local runs